scripts/start.ps1

```diff
@@ -27,11 +27,11 @@ if ($Run) {
     $PARAM_RUN = $true
 }
 
-if (!(Test-Path -Path $ENV:TEMP)) {
-    New-Item -ItemType Directory -Force -Path $ENV:TEMP
-}
+$dateTime = Get-Date -Format "dd-MM-yyyy_HH-mm-ss"
 
-Start-Transcript $ENV:TEMP\Winutil.log -Append
+$logdir = "$env:localappdata\Winutil\Log"
```

Comment:
```suggestion
$logdir = "$env:localappdata\winutil\logs"
```
@MyDrift-user simple suggestion on folder name, as the logic creates multiple log files.. so using `logs` instead of `Log` will make it clear that it could contain multiple files...
Again, it's a simple thing to nitpick, you can ignore it if you want to 👍

Also name of winutil folder is lowercase winutil, check the other occurrences of `localappdata\winutil` ([here's a link to that line](https://github.com/ChrisTitusTech/winutil/blob/3903eaaa24127041896f43a6a5040892f0b8d376/scripts/main.ps1#L427)), idk if WinUtil when checking for and creating folders will it be case sensitive or not (probably not, knowing that Windows is windows)... but just to make sure :)

Reply:
done, thanks :)